specification/servicelinker/resource-manager/Microsoft.ServiceLinker/preview/2023-04-01-preview/servicelinker.json

```diff
@@ -0,0 +1,2806 @@
+          }
+        },
+        "recommendedRole": {
+          "description": "The recommended role to resolve permissions missing",
+          "type": "string"
+        }
+      }
+    },
+    "DryrunOperationPreview": {
+      "description": "The preview of the operations for creation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The operation name",
+          "type": "string"
+        },
+        "operationType": {
+          "description": "The operation type",
+          "type": "string",
+          "enum": [
+            "configConnection",
+            "configNetwork",
+            "configAuth"
+          ],
+          "x-ms-enum": {
+            "name": "DryrunPreviewOperationType",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "description": "The description of the operation",
+          "type": "string"
+        },
+        "action": {
+          "description": "The action defined by RBAC, refer https://docs.microsoft.com/azure/role-based-access-control/role-definitions#actions-format",
+          "type": "string"
+        },
+        "scope": {
+          "description": "The scope of the operation, refer https://docs.microsoft.com/azure/role-based-access-control/scope-overview",
+          "type": "string"
+        }
+      }
+    },
+    "ActionType": {
+      "description": "Indicates how to apply the connector operations, such as opt out network configuration, opt in configuration.",
+      "type": "string",
+      "enum": [
+        "enable",
+        "optOut"
+      ],
+      "x-ms-enum": {
+        "name": "actionType",
+        "modelAsString": true
+      }
+    },
+    "AllowType": {
+      "description": "Whether to allow firewall rules.",
+      "type": "string",
+      "enum": [
+        "true",
+        "false"
+      ],
+      "x-ms-enum": {
+        "name": "allowType",
+        "modelAsString": true
+      }
+    },
+    "DaprConfigurationList": {
+      "description": "Dapr configuration list supported by Service Connector",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of dapr configurations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaprConfigurationResource"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DaprConfigurationResource": {
+      "description": "Represent one resource of the dapr configuration list",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "The properties of the dapr configuration.",
+          "$ref": "#/definitions/DaprConfigurationProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DaprConfigurationProperties": {
```

Comment:
>DaprConfigurationProperties [](http://example.com/codeflow?start=5&length=27)

 MissingTypeObject The schema 'DaprConfigurationProperties' is considered an object but without a 'type:object', please add the missing 'type:object'.
Location: Microsoft.ServiceLinker/preview/2023-04-01-preview/servicelinker.json#L2755

Reply:
[https://github.com/Azure/azure-rest-api-specs/pull/25636/checks?check_run_id=16844703290](https://github.com/Azure/azure-rest-api-specs/pull/25636/checks?check_run_id=16844703290)